galasa-ui/src/utils/3270/get3270Screenshots.ts

```diff
@@ -54,9 +54,9 @@ function unzipBase64(artifactData: {
   return JSON.parse(new TextDecoder().decode(decompressedUint8Array));
 }
 
-export function populateFlattenedZos3270TerminalDataAndAllImageData(images: any[]): void {
-  images.forEach((image: any) => {
-    if (!image.id || !image.fields) {
+export function populateFlattenedZos3270TerminalDataAndAllImageData(images: TerminalImage[]): void {
```

Comment:
Minor - seems a bit strange to have this file called `get3270Screenshots.ts` but we're exporting multiple functions from this file, maybe we should rename this file to be more accurate as to what functions it exports?